packages/grafana-data/src/vector/CircularVector.ts

```diff
@@ -34,6 +36,7 @@ export class CircularVector<T = any> extends FunctionalVector<T> {
     if (options.capacity) {
       this.setCapacity(options.capacity);
     }
+    return makeArrayIndexableVector(this);
```

Comment:
this is the only version that is still in (limited) real use -- the rest can be replaced by arrays easily.